tests/views_integration/Tribe/Events/Views/V2/Partials/List_View/NavTest.php

```diff
@@ -10,20 +10,42 @@ class NavTest extends HtmlPartialTestCase
 	protected $partial_path = 'list/nav';
 
 	/**
-	 * Test static render
-	 * @todo remove this static HTML test once the partial is dynamic.
+	 * Test render with all links
 	 */
-	public function test_static_render() {
-		$this->assertMatchesSnapshot( $this->get_partial_html() );
+	public function test_render_with_all_links() {
+		$this->assertMatchesSnapshot( $this->get_partial_html( [
+			'prev_url'  => '#',
```

Comment:
Same as above, use hard-coded URLs that look like URL to make the snapshot changes evident.